modules/cnf-programming-the-n3000-operator.adoc

```diff
@@ -0,0 +1,348 @@
+----
+
+. Verify the pods are running:
++
+[source,terminal]
+----
+$ oc get pods
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                        READY       STATUS          RESTARTS    AGE
+fpga-driver-daemonset-8xz4c                 1/1         Running         0           15d
+fpgainfo-exporter-vhvdq                     1/1         Running         1           15d
+N3000-controller-manager-b68475c76-gcc6v    2/2         Running         1           15d
+N3000-daemonset-5k55l                       1/1         Running         1           15d
+N3000-discovery-blmjl                       1/1         Running         1           15d
+N3000-discovery-lblh7                       1/1         Running         1           15d
+----
+* `fpgainfo-exporter` provides N3000 telemetry data for Prometheus
+* `fpga-driver-daemonset` provides and loads the required drivers
+* `N3000-controller-manager` applies N3000Node CRs to the cluster
+* `N3000-daemonset` is the main worker application
+* `N3000-discovery` discovers N3000 Accelerator devices installed and labels worker nodes if devices are present.
+
+. Get all the nodes containing the Intel® FPGA PAC N3000 card:
++
+[source,terminal]
+----
+$ oc get n3000node
+----
++
+.Example output
+[source,terminal]
+----
+NAME                       FLASH
+node1                      NotRequested
+----
+
+. Retrieve information about the card on each node:
++
+[source,terminal]
+----
+$ oc get n3000node node1 -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+status:
+  conditions:
+  - lastTransitionTime: "2020-12-15T17:09:26Z"
+    message: Inventory up to date
+    observedGeneration: 1
+    reason: NotRequested
+    status: "False"
+    type: Flashed
+  fortville:
+  - N3000PCI: 0000:1b:00.0
+    NICs:
+    - MAC: 64:4c:36:11:1b:a8
+      NVMVersion: 7.00 0x800052b0 0.0.0
+      PCIAddr: 0000:1a:00.0
+      name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+    - MAC: 64:4c:36:11:1b:a9
+      NVMVersion: 7.00 0x800052b0 0.0.0
+      PCIAddr: 0000:1a:00.1
+      name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+    - MAC: 64:4c:36:11:1b:ac
+      NVMVersion: 7.00 0x800052b0 0.0.0
+      PCIAddr: 0000:1c:00.0
+      name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+    - MAC: 64:4c:36:11:1b:ad
+      NVMVersion: 7.00 0x800052b0 0.0.0
+      PCIAddr: 0000:1c:00.1
+      name: Ethernet Controller XXV710 Intel(R) FPGA Programmable Acceleration Card N3000 for Networking
+  fpga:
+  - PCIAddr: 0000:1b:00.0 <1>
+    bitstreamId: "0x23000410010310" <2>
+    bitstreamVersion: 0.2.3
+    deviceId: "0x0b30"
+----
+<1> PCI Address of card to be programmed
+<2> Currently flashed bitstream
+
+. Save the current `bitstreamId`, `PCIAddr`, the name without "n3000" and the `deviceId` without "0x" padding.
```

Comment:
I think the node name is no longer prefixed with "n3000" so it is unnecessary to mention it? @SchSeba